IfcDistributionElement Rail Import BIMQ 2021-07-21T07:09:54.0010614Z (#171)
bSI-InfraRoom/IFC-Specification: IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_ElectricalDeviceCompliance/Documentation.md

> Properties in this property set are used to define information about compliance to standards or regulations of electric devices.

Reviewer: ```suggestion
Properties related to information about compliance to standards or regulations of electric devices.
```
